### PR TITLE
Metrics in MonitorCrossConnects

### PR DESCRIPTION
**Pull request by denis-tingajkin**
Signed-off-by: denis-tingajkin <denis.tingajkin@xored.com>

## Description

https://github.com/networkservicemesh/networkservicemesh/issues/715
## How Has This Been Tested?

- [ ] Covered by existing integration testing
- [X] Added integration testing to cover
- [X] Tested locally
- [ ] Have not tested

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

**Comment by edwarnicke** [comment]
@matrohon This look good to you for consumption of metrics in skydive?

**Comment by nickolaev** [comment]
Can we also get the draft from #755 in `docs/specs/` with this patch?

**Comment by denis-tingajkin** [comment]
@nickolaev 
I've added simple spec in docs. Please take a look.

File: controlplane/pkg/metrics/metrics_collector.go [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+	"time"
+)
+
+type MetricsCollector struct {
+	requestPeriod time.Duration
+}
+
+func NewMetricsCollector() *MetricsCollector {
+	return &MetricsCollector{
+		requestPeriod: time.Second * 15,
```
Comment (haiodo) [inline controlplane/pkg/metrics/metrics_collector.go]: Should we parametrise this with ENV variable?

Comment (edwarnicke) [reply controlplane/pkg/metrics/metrics_collector.go]: Good idea, but can be done in a followup PR :)

**Pull request merged**